src/styles/app.styl

```diff
@@ -522,6 +529,35 @@ footer
       &[aria-busy=true]::after
         @extend $icon-spinner-blue
 
+.service-success
+    display          none
+    justify-content space-around
+    align-items stretch
+    position         relative
+    background-color transparent
+    color            $grey-09
+    padding          0 (32/base-font-size)em
+    height           (32/base-font-size)em
+
+    .result
+      overflow      hidden
+      text-overflow ellipsis
+
+    .icon-check
+      position            absolute
+      top                 (-2/base-font-size)em
+      right               (8/base-font-size)em
+      width               (24/base-font-size)em
+      height              (24/base-font-size)em
+      border-radius       (12/base-font-size)em
+      background-repeat   no-repeat
+      background-color    #35CE68
+      background-image    embedurl("../../assets/sprites/icon-checked-16.svg");
```

Comment:
Accord to my previous comment, don't forget to change the icon file name here too.